src/integrationTest/java/org/opensearch/security/DlsIntegrationTests.java

```diff
@@ -158,6 +164,56 @@ public class DlsIntegrationTests {
                 .on("*")
         );
 
+    /**
+     * Test role 1 for DLS filtering with two (non)overlapping roles. This role imposes a filter where the user can only access documents where the sensitive field is false. This role is applied at a higher level for all index patterns.
+     */
+    static final TestSecurityConfig.Role TEST_ROLE_ONE = new TestSecurityConfig.Role("test_role_1").clusterPermissions(
```

Comment:
Nit: What do you think about renaming the roles so that they say what they do similar to below with the music tests? I.e ROLE_NON_SENSITIVE_ONLY